README.md

```diff
@@ -1,25 +1,68 @@
-# Percy Report Generator
+# Percy Report Generator <img src="https://files.readme.io/369dd84-logo-dark-icon-32.svg" >
+
 
 ## Installation
 
 ```sh
 
-npm install https://githib.com/browserstackce/percy-report.git -g
+npm install https://github.com/browserstackce/percy-report.git -g
+
+```
+
+<h3> Export Percy Token </h3>
+Get percy Read Only or Full Access token from your percy project settings & export it
 
 ```
+export PERCY_TOKEN=<read-only | full-access>
+```
+
 
 ## Usage
 
-```sh
+<h3>Using the Generate Report functionality independently</h3>
```

Comment:
Dont add "Using" to the sub-headings. Can be something like "Generate Report - Per Build"